### DISCUSSION TITLE
Release binary files missing

### DISCUSSION THREAD

**Discussion by 077uyp** [projectdiscovery/nuclei#1399] [Q&A] [answered]
Why did you delete the binary files?

**Comment by 077uyp** [comment]
[FTL] Could not create runner: no compatible binary found for linux/amd64

**Comment by ehsandeep** [comment]
@077uyp there is build fail https://github.com/projectdiscovery/nuclei/actions/runs/1596210411 which results in no binary files are generated, additionally, you can update to the latest version with `go install -v github.com/projectdiscovery/nuclei/v2/cmd/nuclei@latest`

also, use discussion instead of issues for asking questions.

**Comment by ehsandeep** [comment, marked as answer]
should be fixed with latest release
